src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -613,6 +613,26 @@ End Module]]></Document>)
             End Using
         End Sub
 
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
WorkItem(###)? Do we have a convention for tagging tests with github workitem numbers that distinguishes from TFS workitem numbers?